qiita_db/test/test_analysis.py

```diff
@@ -95,12 +100,12 @@ def test_create(self):
 
         new = Analysis.create(User("admin@foo.bar"), "newAnalysis",
                               "A New Analysis")
-        self.assertEqual(new.id, 3)
+        self.assertEqual(new.id, 7)
```

Comment:
it's going to be hard to determine what the other magic numbers are anyways
so I think working to reduce this burden overall is in everyones benefit

On Thu, Apr 2, 2015 at 4:57 PM, josenavas <notifications@github.com> wrote:

> In qiita_db/test/test_analysis.py
> <https://github.com/biocore/qiita/pull/1025#discussion_r27704832>:
>
> > @@ -95,12 +100,12 @@ def test_create(self):
> >
> >          new = Analysis.create(User("admin@foo.bar"), "newAnalysis",
> >                                "A New Analysis")
> > -        self.assertEqual(new.id, 3)
> > +        self.assertEqual(new.id, 7)
>
> Oh no, this is something that we are trying to remove, and @wasade
> <https://github.com/wasade> has done strict comments on changing this,
> and I thing all @biocore/qiita-dev
> <https://github.com/orgs/biocore/teams/qiita-dev> agree that this has to
> be done so we do not spend more time fixing magic numbers...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/biocore/qiita/pull/1025/files#r27704832>.
>